.buildkite/common.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+Common helpers to create Buildkite pipelines
+"""
+
+DEFAULT_INSTANCES = [
+    "m5d.metal",
+    "m6i.metal",
+    "m6a.metal",
+    "m6g.metal",
+]
+
+DEFAULT_KERNELS = ["linux_4.14", "linux_5.10"]
+
+
+def group(label, command, instances, kernels, agent_tags=None, **kwargs):
+    """
+    Generate a group step with specified parameters, for each instance+kernel
+    combination
+
+    https://buildkite.com/docs/pipelines/group-step
+    """
+    if agent_tags is None:
+        agent_tags = []
```

Comment:
Would it be simpler to include `[]` in the definition?
```suggestion
def group(label, command, instances, kernels, agent_tags=[], **kwargs):
    """
    Generate a group step with specified parameters, for each instance+kernel
    combination
    https://buildkite.com/docs/pipelines/group-step
    """
```

Reply:
That's one of Python's warts: https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/